src/services/WolfDen.Domain/Entity/LOP.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace WolfDen.Domain.Entity
+{
+    public class LOP
+    {
+        public int Id { get; private set; }
+        public int Month { get;private set; }
+        public int EmployeeId { get; private set; }
+        public int LOPDays { get; private set; }

```

Comment:
What does this indicate

Reply:
to store the no of loss of pay days in a month for each employee for the monthly report generation after attendence close